src/commons/components/catalog-content/catalog-content.tsx

```diff
@@ -7,7 +7,21 @@ import CatalogContentItem from '../catalog-content-item';
 import './catalog-content.css';
 import './catalog-content-media.css';
 
-function CatalogContent(props) {   
+
+interface CatalogContentProps { 
+  objMelodies: Array<CatalogContentItemPropsMelody>
```

Comment:
it is better to use syntax like this

objMelodies: CatalogContentItemPropsMelody[ ]

why is it called objMelodies if inside array?